h2o-py/h2o/frame.py

```diff
@@ -448,15 +448,17 @@ def show(self, use_pandas=False, rows=10, cols=200):
             print("This H2OFrame is empty.")
             return
         if not self._ex._cache.is_valid(): self._frame()._ex._cache.fill()
-        if H2ODisplay._in_ipy():
+        if H2ODisplay._in_zep():
+            print("%html " + self._ex._cache._tabulate("html", False, rows=rows))
+        elif H2ODisplay._in_ipy():
             import IPython.display
             if use_pandas and can_use_pandas():
-                IPython.display.display(self.head(rows=rows, cols=cols).as_data_frame(fill_cache=True))
+                IPython.display.display(self.head(rows=rows, cols=cols).as_data_frame(use_pandas=True))
```

Comment:
What is please reason to remove`fill_cache` and introduce`use_pandas`?

Reply:
because as_data_frame has no argument fill_cache and this caused an error